desc/protoparse/parser.go

```diff
@@ -387,16 +395,16 @@ func fixupFilenames(protos map[string]*parseResult) map[string]*parseResult {
 	return revisedProtos
 }
 
-func parseProtoFiles(acc FileAccessor, filenames []string, errs *errorHandler, recursive, validate bool, parsed *parseResults) {
+func (p Parser) parseProtoFiles(acc FileAccessor, filenames []string, errs *errorHandler, recursive, validate bool, parsed *parseResults) {
```

Comment:
We should either (a) use a pointer receiver so we don't have to copy the entire struct on the stack or (b) just add a singular new argument to convey the dependency descriptors. Passing `p` as the receiver just to avoid an explicit argument seems really wasteful since `p` has so many other things that we don't care about.

My personal preference would probably be the second option, just extend the argument list.

Reply:
Ack. Will do.

Reply:
Done.